docs/tutorials/best-practices/error-safety.md

```diff
@@ -252,7 +252,7 @@ consider this next snippet. You can write it using `Safe`, `xpcall` and `pcall`
 === "Safe"
 
 	```Lua
-	local double = scope:Computed(function()
+	local double = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
	local double = scope:Computed(function(use)
```

Reply:
removed